lib/plugins/aws/deploy/compile/events/apiGateway/tests/validate.js

```diff
@@ -67,4 +67,43 @@ describe('#validate()', () => {
 
     expect(() => awsCompileApigEvents.validate()).to.throw(Error);
   });
+
+  it('should validate the http events object syntax method is case insensitive', () => {
+    awsCompileApigEvents.serverless.service.functions = {
+      first: {
+        events: [
+          {
+            http: {
+              method: 'POST',
+              path: 'foo/bar',
+            },
+          },
+          {
+            http: {
+              method: 'post',
+              path: 'foo/bar',
+            },
+          },
+        ],
+      },
+    };
+
+    expect(() => awsCompileApigEvents.validate()).to.not.throw(Error);
+  });
```

Comment:
Would love one newline here. But we could add this in a cleanup PR when we audit the whole codebase.